### ISSUE TITLE
Enhancement of session class

### ISSUE THREAD

**Issue by nggary** [gunthercox/ChatterBot#942]
Hi there, I am working on a way to customize the intention of a conversation seesion, It world be nice to have extra storage variable in session object, so that make each conversation has its own intention
Currently there are only uuid, id_string, id included in session class:
http://chatterbot.readthedocs.io/en/stable/_modules/chatterbot/conversation/session.html#Session

**Comment by gunthercox** [comment]
Hi @nggary some changes have recently been made and the session object is being removed in the next release. It was replaced with a "conversation" object which is persisted in the database.

Additionally I opened issue #904 which, once completed, will make it possible for the existing objects such as statements, responses and conversations to be overwritten so that additional method and variables can be added to them.

I agree with all of the functionality you are asking for, this is a question that has come up on multiple occasions. Let me know what you think of these changes. I'm hoping that they will make it much easier to accomplish what you want.
